lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -261,10 +261,11 @@ defmodule ExUnit.DocTest do
           reraise e, stack
```

Comment:
I don't think this branch can even be invoked. I.e. people shouldn't be using `assert` in their doc tests. I think we can safely remove this?